onnx/defs/logical/defs.cc

```diff
@@ -121,6 +121,34 @@ ONNX_OPERATOR_SET_SCHEMA(
             {"tensor(bool)"},
             "Constrains output to boolean tensor."));
 
+ONNX_OPERATOR_SET_SCHEMA(
+	LessOrEqual,
+	9,
+	OpSchema()
+		.FillUsing(BinaryLogicDocGenerator("less or equal"))
+		.TypeConstraint(
+			"T",
+			OpSchema::all_numeric_types(),
+			"Constarins input to float tensors.")
+		.TypeConstraint(
+			"T1",
+			{"tensor(bool)"},
+			"Constrains output to boolean tensor."));
+
+ONNX_OPERATOR_SET_SCHEMA(
+	GreaterOrEqual,
+	9,
+	OpSchema()
+		.FillUsing(BinaryLogicDocGenerator("greater or equal"))
+		.TypeConstraint(
+			"T",
+			OpSchema::all_numeric_types(),
+			"Constarins input to float tensors.")
+		.TypeConstraint(
+			"T1",
+			{"tensor(bool)"},
+			"Constrains output to boolean tensor."));
```

Comment:
Maybe consider adding type and shape inference function as they are increasingly being used by backends for model validation.